ansible/roles/contiv/tasks/restart_netplugin.yml

```diff
@@ -0,0 +1,71 @@
+# stop services
```

Comment:
same with restart_netmaster.yml. It would be really nice if we can get away from having to duplicate all the code in the start netplugin/master just to restart the services.